src/main.rs

```diff
@@ -20,5 +30,90 @@ impl edg::ExtendedDependencyGraph<i32> for EmptyGraph {
 }
 
 fn main() {
-    edg::distributed_certain_zero(EmptyGraph {}, 0, num_cpus::get() as u64);
+    let args = parse();
+
+    // Load config for logging to stdout and logfile.
+    if let Ok(_handle) = log4rs::init_config(get_log4rs_config(
+        args.value_of("log-path").unwrap(),
+        match args.value_of("log-level").unwrap().to_lowercase().as_str() {
+            "error" => LevelFilter::Error,
+            "warn" => LevelFilter::Warn,
+            "info" => LevelFilter::Info,
+            "debug" => LevelFilter::Debug,
+            "trace" => LevelFilter::Trace,
+            "off" => LevelFilter::Off,
+            _ => LevelFilter::Off,
```

Comment:
We should probably fail with an informative error message instead of silently accepting wrong input

Reply:
Good idea, applied in d69b7c2